From 1e895eb6fdbc20925ef0a54272adf5b02a961dff Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Mon, 12 Aug 2013 14:43:48 -0700
Subject: x86: compile C files based on TARGET_ARCH

TARGET_ARCH_VARIANT is used in the build to distinguish between
different kinds of x86 processors (atom, ivy bridge, etc.) For
the paths referenced there is only 'x86', so use TARGET_ARCH
instead.

Bug: ADEV-107
Category: Device Enablement
Domain: AOSP-bionic
Origin: Internal
Upstream-candidate: Yes

Change-Id: I3add06082537af00ac980b00f2c81e070b86af00
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 vm/Dvm.mk |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/vm/Dvm.mk b/vm/Dvm.mk
index 2af0582..6e398b7 100644
--- a/vm/Dvm.mk
+++ b/vm/Dvm.mk
@@ -292,10 +292,10 @@ ifeq ($(dvm_arch),x86)
     LOCAL_CFLAGS += -DDVM_JMP_TABLE_MTERP=1 \
                     -DMTERP_STUB
     LOCAL_SRC_FILES += \
-		arch/$(dvm_arch_variant)/Call386ABI.S \
-		arch/$(dvm_arch_variant)/Hints386ABI.cpp \
-		mterp/out/InterpC-$(dvm_arch_variant).cpp \
-		mterp/out/InterpAsm-$(dvm_arch_variant).S
+		arch/$(dvm_arch)/Call386ABI.S \
+		arch/$(dvm_arch)/Hints386ABI.cpp \
+		mterp/out/InterpC-$(dvm_arch).cpp \
+		mterp/out/InterpAsm-$(dvm_arch).S
     ifeq ($(WITH_JIT),true)
       LOCAL_CFLAGS += -DARCH_IA32
       LOCAL_SRC_FILES += \
-- 
1.7.4.1

